keanu-python/tests/test_plates.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, Dict
+
+import pytest
+
+from keanu.plates import Plates, Plate
+from keanu.vertex import Bernoulli, DoubleProxy, Exponential, Poisson, Const, KeanuContext
+
+
+def test_you_can_iterate_over_the_plates() -> None:
+    num_plates = 100
+
+    plates = Plates(count=num_plates, factory=lambda p: None)
+    plate_count = 0
+    for plate in plates:
```

Comment:
nit:

sum(1 for plate in plates) seems a bit more pythonic?

Reply:
nice!